README.md

```diff
@@ -95,10 +111,12 @@ Run `make help` to see all the possible targets and variables:
     unicharset       Create unicharset
     charfreq         Show character histogram
     lists            Create lists of lstmf filenames for training and eval
-    training         Start training
+    training         Start training (i.e. create .checkpoint files)
```

Comment:
That additional information is an internal detail (which might change in the future). I suggest to keep the old text.

Reply:
The descriptions of `traineddata` and (new) `lstmeval` already refer to it, though.